PWGCF/EbyEFluctuations/Tasks/MeanPtFlucIdentified.cxx

```diff
@@ -47,12 +47,14 @@ struct meanPtFlucId {
   Configurable<int> nPartBins{"nPartBins", 250, ""};
   Configurable<int> nCentBins{"nCentBins", 101, ""};
   Configurable<int> nEtaBins{"nEtaBins", 100, ""};
+  Configurable<int> nTpNBins{"nTpNBins", 200, ""};
+  Configurable<int> nTpDBins{"nTpDBins", 100, ""};
   Configurable<float> cfgCutPtMax{"cfgCutPtMax", 2.0, "maximum pT"};
   Configurable<float> cfgCutPtMin{"cfgCutPtMin", 0.15, "minimum pT"};
   Configurable<float> cfgCutEta{"cfgCutEta", 0.8, "Eta cut"};
   Configurable<float> cfgCutRap{"cfgCutRap", 0.5, "Rapidity Cut"};
   Configurable<float> cfgCutDcaXY{"cfgCutDcaXY", 0.12, "DCAxy cut"};
-  Configurable<float> cfgCutDcaZ{"cfgCutDcaZ", 1.0, "DCAz cut"};
+  Configurable<float> cfgCutDcaZ{"cfgCutDcaZ", 0.3, "DCAz cut"};
```

Comment:
Remember these are configurables so you don't need to change their value in the source code, you can do it by configuration